purple-pill/js/metapolatorStandAlone.js

```diff
@@ -39713,10 +39967,6 @@ function (
                  : resolve()
                  ;
         }
-      , cpsGenerators: {
```

Comment:
should be gone now, I just pushed to https://github.com/metapolator/metapolator/blob/gh-pages/purple-pill/js/metapolatorStandAlone.js

I had the same diff: https://github.com/metapolator/metapolator/commit/d4f924274f1ea35ed4a1618f7afc35fb552f6aab#diff-b17319f6c1b99e02b6d184f1e80feed9L39716

This was just a left over, as explained above.